sweetest/src/main/java/com/mysugr/sweetest/framework/context/ConfigurationsTestContext.kt

```diff
@@ -2,16 +2,18 @@ package com.mysugr.sweetest.framework.context
 
 import com.mysugr.sweetest.framework.configuration.ModuleTestingConfiguration
 
-internal class ConfigurationsTestContext(private val factories: FactoriesTestContext) {
+class ConfigurationsTestContext(private val factories: FactoriesTestContext) {
 
     private val moduleConfigurations = mutableListOf<ModuleTestingConfiguration>()
 
     val all = moduleConfigurations as List<ModuleTestingConfiguration>
 
-    fun put(configuration: ModuleTestingConfiguration) {
+    fun put(configuration: ModuleTestingConfiguration?) {
```

Comment:
hm, somehow it seems to me that it makes no real sense to pass `null` here. can't we just avoid the call if it's null? 🤔